db/seeds/installations.json

```diff
@@ -0,0 +1 @@
"011-81-98-961-3366","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"011-81-98-961-0539","phone_type":"fax","dsn":false,"note":"Commercial Fax"},{"phone_number":"315-634-3366","phone_type":"voice","dsn":true,"note":"DSN Phone"},{"phone_number":"315-634-0539","phone_type":"fax","dsn":true,"note":"DSN Fax"}],"urls":[{"url":"http://www.kadena.af.mil","note":"Website Address"}]},{"name":"Keesler AFB","service_name":"Air Force","service_code":"usaf","street_address":"211 Larcher Blvd","extended_address":"Building 4605","locality":"Biloxi","region":"Mississippi","region_code":"MS","postal_code":"39534","country_name":"United States","country_code":"US","latitude":30.3986694,"longitude":-88.91108559999999,"phone_numbers":[{"phone_number":"228-376-8728","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"1-866-301-9436","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"228-377-3063","phone_type":"fax","dsn":false,"note":"Commercial Fax"},{"phone_number":"312-591-8728","phone_type":"voice","dsn":true,"note":"DSN Phone"},{"phone_number":"312-597-3063","phone_type":"fax","dsn":true,"note":"DSN Fax"}],"urls":[{"url":"http://www.keesler.af.mil","note":"Website Address"}]},{"name":"Kirtland Air Force Base","service_name":"Air Force","service_code":"usaf","street_address":"2000 Wyoming Blvd SE","extended_address":"Consolidated Support Building 20245","locality":"Albuquerque","region":"New Mexico","region_code":"NM","postal_code":"87123","country_name":"United States","country_code":"US","latitude":35.0359909,"longitude":-106.5505184,"phone_numbers":[{"phone_number":"505-846-0741","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"505-846-0751","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"505-846-9889","phone_type":"fax","dsn":false,"note":"Commercial Fax"},{"phone_number":"312-246-0741/0751","phone_type":"voice","dsn":true,"note":"DSN Phone"},{"phone_number":"312-246-9889","phone_type":"fax","dsn":true,"note":"DSN Fax"}],"urls":[{"url":"https://www.kirtland.af.mil","note":"Website Address"}]},{"name":"Kleine Brogel Air Base","service_name":"Air Force","service_code":"usaf","street_address":"Onze-Lieve-Vrouwlaan","extended_address":null,"locality":"Peer","region":"Vlaanderen","region_code":"Vlaanderen","postal_code":"3990","country_name":"Belgium","country_code":"BE","latitude":51.169724,"longitude":5.471110899999999,"phone_numbers":null,"urls":null},{"name":"Kunsan AB","service_name":"Air Force","service_code":"usaf","street_address":"Gunsan Airport","extended_address":"Building 755","locality":"Gunsan","region":"Jeollabuk-do","region_code":"Jeollabuk-do","postal_code":"573-950","country_name":"South Korea","country_code":"KR","latitude":35.9259206,"longitude":126.6156607,"phone_numbers":[{"phone_number":"011-82-63-470-5644","phone_type":"voice","dsn":false,"note":"Commercial Phone"},{"phone_number":"011-82-63-470-4019","pho
```

Comment:
Is it too much to ask for some newlines and indentation in this file? It's not going to be sent over the wire, so size is not a concern.